### PR TITLE
ARMOCP-46: Adding ARM RDU2 Bare Metal IaaS

### PR DESCRIPTION
**Pull request by aleskandro**
Tested by https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/71100/

### CODE REVIEW

**Comment by aleskandro** [comment]
Refers https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/merge_requests/906

File: lib/launchers/armrdu2.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+lib_path = File.expand_path(File.dirname(File.dirname(__FILE__)))
+unless $LOAD_PATH.any? {|p| File.expand_path(p) == lib_path}
+  $LOAD_PATH.unshift(lib_path)
+end
+
+require 'collections'
+require 'common'
+
+module BushSlicer
+  class ArmRdu2
```
Comment (pruan-rht) [inline lib/launchers/armrdu2.rb:10]: @aleskandro I suggest we follow the current class naming scheme (please refer to [OCMCluster class](https://github.com/openshift/verification-tests/blob/master/lib/launchers/o_c_m_cluster.rb#L17) for reference), so this should become `ARMRdu2` and the file name becomes `a_r_m_rdu2.rb`

Comment (aleskandro) [reply lib/launchers/armrdu2.rb]: Done @pruan-rht. thanks.

**Comment by aleskandro** [comment]
Refers ARMOCP-62

**Pull request merged**